com.slack.Slack.json

```
@@ -59,7 +60,7 @@
                     "type": "script",
                     "dest-filename": "slack.sh",
                     "commands": [
-                        "if [ -z \"$DISPLAY\" ] && [ -n \"$WAYLAND_DISPLAY\" ]; then",
+                        "if [ \"${XDG_SESSION_TYPE}\" = \"wayland\" ]; then",
```

We only enable wayland if it has been explicitly enabled (via overrides) by the user.

That said, perhaps this approach would be acceptable?

```suggestion
                        "if [ \"${XDG_SESSION_TYPE}\" = \"wayland\" ] && [ -n \"$WAYLAND_DISPLAY\" ]; then",
```